pillar.example

```diff
@@ -12,6 +12,9 @@ hana:
         software_path: '/root/sap_inst/51052481'
         root_user: 'root'
         root_password: 's'
+        # Fetch HANA passwords from XML file 
+        hdb_pwd_file: '/root/passwords.xml'
```

Comment:
As the password must be obtained from the salt master file directory, I would change the example to something like:
`hdb_pwd_file: 'salt://passwords.xml'`